### PR TITLE
feat(route): add 太平洋科技

### PR DESCRIPTION
**Pull request by CH563**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/pconline/focus
/pconline/focus/tech
/pconline/focus/finance
/pconline/focus/life
/pconline/focus/company
/pconline/focus/character
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

支持太平洋科技网要科技新闻栏目的订阅。

提供每日科技要闻，科技新闻快讯，包括科技财经新闻，科技生活新闻，科技公司新闻热点等，并对行业进行深度剖析，实时为您提供每天科技热点，助您快速了解行业信息。

栏目地址：https://www.pconline.com.cn/focus/

### CODE REVIEW

File: lib/routes/pconline/focus.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                },
+            ],
+            pubDate: parseDate(item.pc_pubDate),
+            link: item.url,
+            description: item.summary,
+            category: item.channelName,
+            image: item.cover,
+        }));
+        await Promise.all(items.map((item) => getContent(item)));
+        cache.set(currentUrl, JSON.stringify(items));
```
Comment (TonyRL) [inline lib/routes/pconline/focus.ts:108]: https://docs.rsshub.app/joinus/advanced/use-cache#cache-tryget-key-getvaluefunc-maxage-refresh

> You should use `cache.tryGet()` most of the time.

Comment (CH563) [reply lib/routes/pconline/focus.ts:108]: OK，Thanks you

File: lib/routes/pconline/focus.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        item.description = $('.context-box .context-table tbody td').html();
+        return item;
+    });
+
+export const handler = async (ctx) => {
+    const { category = 'all' } = ctx.req.param();
+    const cate = categories[category] || categories.all;
+    const currentUrl = `${rootUrl}/3g/other/focus/${cate.path}index.html`;
+
+    const items = await cache.tryGet(currentUrl, async () => {
```
[the remark this answers is not in the document]
Comment (CH563) [reply lib/routes/pconline/focus.ts]: 👌

File: lib/routes/pconline/focus.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        category: item.channelName,
+        image: item.cover,
+    }));
+
+    await Promise.all(items.map((item) => getContent(item)));
+
+    return {
+        title: `太平洋科技-${cate.title}`,
+        link: currentUrl,
+        item: items as Item[],
```
Comment (TonyRL) [inline lib/routes/pconline/focus.ts:107]: The promise results should be returned instead of `items`

Comment (CH563) [reply lib/routes/pconline/focus.ts]: 👌

File: lib/routes/pconline/focus.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                    name: item.authorname,
+                    avatar: item.authorImg,
+                },
+            ],
+            pubDate: parseDate(item.pc_pubDate),
+            link: item.url,
+            description: content || item.summary,
+            category: item.channelName,
+            image: item.cover,
+        };
```
Comment (TonyRL) [inline lib/routes/pconline/focus.ts:101]: Please cache the whole returned object instead of `content` only

Comment (CH563) [reply lib/routes/pconline/focus.ts:101]: sorry, I don't understand the return object, `content` is the entire article HTML

Comment (TonyRL) [reply lib/routes/pconline/focus.ts:101]: > whole returned object
```js
return {
    id: ...,
    title: ...,
    ...
}
```

Comment (CH563) [reply lib/routes/pconline/focus.ts]: Okay, thank you.

**Pull request merged**